src/ServicePulse.Host/vue/src/composables/serviceServiceControlUrls.js

```diff
@@ -80,19 +80,25 @@ export function useServiceControlUrls() {
 }
 
 export function updateServiceControlUrls(newServiceControlUrl, newMonitoringUrl) {
-  if (!newServiceControlUrl) {
+  const regex = /\/$/gm;  // checks for a trailing forward slash
```

Comment:
Do we really need regex for this? Can't a simple if check suffice?

I.e. `if (!newServiceControl.value.endsWith('/'))`